Source/ActivityIndicatorView.swift

```diff
@@ -27,7 +27,7 @@ public struct ActivityIndicatorView: View {
     var type: IndicatorType
 
     @State private var isAnimating: Bool = false
```

Comment:
Is this one still used? If not, doesn't it change the way it works for user?

Reply:
not used anymore, will be removed. it shouldn't affect the user, the user works with `isVisible` prop. but I will check it anyway later, thanks!